e2e/features/record-hearing-attendees-and-duration.feature

```diff
@@ -60,3 +60,12 @@ Feature: Record hearing attendees and duration
 
```

Comment:
We can add a story number as Tag on top of the scenario for example @RIA-1903, same for Case-progression as well